slang_frontend.cc

```diff
@@ -820,6 +820,8 @@ struct ProceduralVisitor : public ast::ASTVisitor<ProceduralVisitor, true, false
 		return ret;
 	}
 
+	std::vector<RTLIL::Wire*> disableLoop_ins, disableLoop_outs;
```

Comment:
I think this will crash/miscompile if you have two nested for loops.

Reply:
Right! I'm thinking of making the push/pop_frame function save these wires in a Frame to avoid it. What do you think?

Reply:
Saving this in a `Frame` sounds good. I suggest the `push_frame` function takes an optional "disable wire" argument.

Reply:
I just added the extra parameter in push_frame